src/dts/index.d.ts

```diff
@@ -6,3 +6,6 @@
 /// <reference path="signals.d.ts" />
 /// <reference path="misc.d.ts" />
 /// <reference path="mithril.d.ts" />
+
```

Comment:
node types are not used so this should be removed

Reply:
Whoops. fixed